thewalrus/symplectic.py

```diff
@@ -171,6 +171,9 @@ def reduced_state(mu, cov, modes):
     """
     N = len(mu) // 2
 
+    if type(modes) == np.ndarray:
```

Comment:
You probably need to add a test to trigger this line.

Reply:
Thank you. I added the test!